### PR TITLE
SearchBar: add search bar template

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1585
[Problem] Search bar template is not implemented
[Solution]
   - new template has been added
   - change style on focus
   - open history

`@todo` other issues:
- add microphone icon (https://github.com/Samsung/TAU/issues/1604)
- open popup with results (https://github.com/Samsung/TAU/issues/1605)

[Screenshot]
image
image
image
image
image

    
Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/search/search-bar.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+	function onPageShow() {
+		search = page.querySelector("header .ui-appbar-title-container input[type='search']");
+		search.addEventListener("focus", onSearchFocus);
+	}
+
+	function onPageHide(ev) {
+		var page = ev.target;
+
+		page.removeEventListener("pageshow", onPageShow);
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/search/search-bar.js:20]: How about adding the removeEventListener for the focus event.

Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/search/search-bar.js:20]: You are right. I'll fix it.

Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/search/search-bar.js:20]: Done.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Comment by TomaszLukawskiSam** [comment]
The PR has been rebased.

**Pull request merged**